src/main/java/uk/gov/companieshouse/insolvency/data/api/InsolvencyApiService.java

```diff
@@ -54,19 +54,13 @@ CHANGED_RESOURCE_URI, mapChangedResource(
                                 contextId, insolvencyDocument, eventType)
                 );
         try {
+            LOGGER.info("Calling CHS Kafka API", DataMapHolder.getLogMap());
             return changedResourcePost.execute();
-        } catch (ApiErrorResponseException exp) {
-            HttpStatus statusCode = HttpStatus.valueOf(exp.getStatusCode());
-            if (!statusCode.is2xxSuccessful() && statusCode != HttpStatus.SERVICE_UNAVAILABLE) {
-                logger.error("Unsuccessful call to /private/resource-changed endpoint", exp);
-                throw new MethodNotAllowedException(exp.getMessage());
-            } else if (statusCode == HttpStatus.SERVICE_UNAVAILABLE) {
-                logger.error("Service unavailable while calling /private/resource-changed endpoint", exp);
-                throw new ServiceUnavailableException(exp.getMessage());
-            } else {
-                logger.error("Error occurred while calling /private/resource-changed endpoint", exp);
-                throw new RuntimeException(exp);
-            }
+        } catch (ApiErrorResponseException ex) {
+            DataMapHolder.get().status(Integer.toString(ex.getStatusCode()));
```

Comment:
I am not sure we should set the `status` field in the `DataMap` as this will persist for the rest of the service call. A solution us to reset the `status` field to null after the log line